classes/filter/Filter.inc.php

```diff
@@ -306,7 +306,7 @@ public function clearErrors()
      */
     public function setRuntimeEnvironment(&$runtimeEnvironment)
     {
-        assert($runtimeEnvironment instanceof RuntimeEnvironment);
+        assert(is_a($runtimeEnvironment, 'RuntimeEnvironment'));
```

Comment:
I'm not sure where this would have come from, but I remember from a previous dev meeting that `instance` was a preferable alternative to `is_a`. Not sure if that's also the case here. Is it possible this snuck in from a messy merge conflict/rebase?

Reply:
Yeah, not sure. I'd say unless you know a reason for it to be changed, set it back to how it was.